contrib/babelfishpg_tsql/sql/object_definition_tsql.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE sys.babelfish_view_def (
+	dbid SMALLINT NOT NULL,
+	schema_name NAME NOT NULL,
+	object_name NAME NOT NULL,
+	definition TEXT NOT NULL COLLATE "C",
```

Comment:
Don't we need to capture session property (i.e. ANSI_NULLS)? In my short experiment, select on SQL view follows the setting at when it is created.

1> create table tt1(a varchar(10));
2> go
1> insert into tt1 values (null);
2> go
1> set ansi_nulls off;
2> go
1> select * from tt1 where a = null;
2> go
a
----------
NULL
1> create view v11 as select * from tt1 where a = null;
2> go
1> select * from v11;
2> go
a
----------
NULL
1> set ansi_nulls on;
2> go
1> select * from v11;
2> go
a
----------
NULL

(1 rows affected)
1> select * from tt1 where a = null;
2> go
a
----------

(0 rows affected)

Reply:
Yes, we should add session properties like ANSI_NULLS, QUOTED_IDENTIFIER. I will address this in upcoming commits.